### PR TITLE
feat: CUDOS-949 Implement database migration logic

### PR DESCRIPTION
**Pull request by avalkov**

### CODE REVIEW

File: cmd/migrate-db/migrateDB.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	"github.com/forbole/juno/v2/cmd/parse"
+	"github.com/spf13/cobra"
+)
+
+// NewMigrateDBCmd returns the Cobra command allowing to migrate the db up to latest scheme
+func NewMigrateDBCmd(parseCfg *parse.Config) *cobra.Command {
+	return &cobra.Command{
+		Use:     "migrate-db",
+		Short:   "Migrates database to latest scheme from database/scheme folder",
+		Example: "bdjuno migrate-db",
```
Comment (mlukanova) [inline cmd/migrate-db/migrateDB.go]: I would suggest changing the name of the command to `database migrate`

Comment (avalkov) [reply cmd/migrate-db/migrateDB.go]: Fixed

File: cmd/database-migrate/database-migrate.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	"fmt"
+	"time"
+
+	"github.com/forbole/bdjuno/v2/database"
+	"github.com/forbole/juno/v2/cmd/parse"
+	"github.com/spf13/cobra"
+)
+
+// NewDatabaseMigrateCmd returns the Cobra command allowing to migrate the db up to latest scheme
+func NewDatabaseMigrateCmd(parseCfg *parse.Config) *cobra.Command {
```
Comment (avalkov) [inline cmd/database-migrate/database-migrate.go:14]: Docker files should be modified to execute the command

Comment (avalkov) [reply cmd/database-migrate/database-migrate.go:14]: Fixed.

**Review by SpaghettiOverload — APPROVED** [review]

**Pull request merged**